File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int counter = 1;
        int scan = 1;
        while(scan < nums.size()){
            if(nums[scan-1] != nums[scan]){
                nums[counter] = nums[scan];
                scan++;
                counter++;
            }
            else{
                scan++;
            }
            
        }
        return counter;
}
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        count = 1;
        scanner = 1;
        while(scanner < len(nums)):
            if nums[scanner-1] != nums[scanner]:
                nums[count] = nums[scanner]
                count+=1
                scanner+=1
            else:
                scanner+=1
        return count